### PR TITLE
126 create placeholder mic for problem page

### PR DESCRIPTION
**Pull request by dpmacisaac**
Testing: None at the moment since its a placeholder
Documentation: Added comments on the pages edited and added.

### CODE REVIEW

File: ComSemApp/models.py [hunk 1]
```diff
@@ -377,3 +377,4 @@ class ExpressionError(models.Model):
     end_index = models.IntegerField(null=True)
     # the user who tagged the error
     user = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
+
```
Comment (aedmondson) [inline ComSemApp/models.py:380]: Try not to make whitespace changes in places you haven't changed otherwise

File: ComSemApp/student/views.py [hunk 2]
```diff
@@ -670,7 +670,7 @@ def form_valid(self, form):
 # the foo() functions are just placeholders too keep the classes in here
 class SpeakingPracticeView(StudentViewMixin, CourseViewMixin, TemplateView):
     template_name: str = 'ComSemApp/student/assessment.html'
-
+
```
Comment (aedmondson) [inline ComSemApp/student/views.py:673]: Another whitespace change that should be reverted

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,34 @@
+asgiref==3.5.2
```
Comment (aedmondson) [inline requirements.txt:1]: Not sure if we should be adding the requirements.txt

**Review by aedmondson — CHANGES_REQUESTED** [review]
Whitespace issues and requirements.txt need looked at

File: requirements.txt (hunk 3 above)
Comment (dpmacisaac) [reply requirements.txt]: fixed the white space issues.
Do you know how to ignore that requirements.txt file so we can go ahead and merge this in?

Comment (dpmacisaac) [reply requirements.txt]: Went ahead and just removed it for now.

**Pull request merged**